packages/ses/src/get-anonymous-intrinsics.js

```diff
@@ -48,19 +52,19 @@ export const getAnonymousIntrinsics = () => {
   // 21.1.5.2 The %StringIteratorPrototype% Object
 
   // eslint-disable-next-line no-new-wrappers
-  const StringIteratorObject = new String()[iteratorSymbol]();
+  const StringIteratorObject = iterateString(new String());
   const StringIteratorPrototype = getPrototypeOf(StringIteratorObject);
 
   // 21.2.7.1 The %RegExpStringIteratorPrototype% Object
   const RegExpStringIterator =
-    RegExp.prototype[matchAllSymbol] && new RegExp()[matchAllSymbol]();
+    regexpPrototype[matchAllSymbol] && matchAllRegExp(new FERAL_REG_EXP());
```

Comment:
Could this one be
```suggestion
    regexpPrototype[matchAllSymbol] && matchAllRegExp(/./);
```
?